src/components/LazyLog/index.jsx

```diff
@@ -277,8 +296,8 @@ export default class LazyLog extends Component {
     });
   };
 
-  handleEnd = () => {
-    this.setState({ loaded: true });
+  handleEnd = encodedLog => {
+    this.setState({ loaded: true, encodedLog });
```

Comment:
Thoughts on removing the encoded log from state? I don't think the value ever changes again, and so would never affect the need to re-render.

```js
handleEnd = encodedLog => {
  this.encodedLog = encodedLog;
  this.setState({ loaded: true 
```